Update documentation for distributed environment (#7370)
wazuh/wazuh-documentation: source/user-manual/user-administration/password-management.rst

If you want to change the password for a Wazuh manager API user, run the script on a Wazuh server node and use option ``-A, --api``. Alternatively, you can change the Wazuh manager API passwords following the instructions in the :doc:`Securing the Wazuh API </user-manual/api/securing-api>` documentation.

> .. note:: If you want to change the password for Filebeat in the Wazuh server, you don't need to use option ``-A, --api``.

Reviewer: @Enaraque More useful would be to say what to do rather than what not to do. And I don't see the need to use a note here. Just a paragraph like the previous ones would be ok.

    INFO: The password for user snapshotrestore is Z6UXgM8Sr0bfV.i*6yPPEUY3H6Du2rdz
    WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, and Filebeat nodes if necessary, and restart the services.

If you use the tool in an all-in-one deployment, it automatically updates the passwords where necessary, including the Filebeat password in the Wazuh server:

  .. code-block:: console
    :class: output
    :emphasize-lines: 2,3,4

    INFO: Wazuh API admin credentials not provided, Wazuh API passwords not changed.
    INFO: The new password for Filebeat is kwd139yG?YoIK?lRnqcXQ4R4gJDlAqKn
    INFO: The password for user admin is kwd139yG?YoIK?lRnqcXQ4R4gJDlAqKn
    INFO: The password for user kibanaserver is Bu1WIELh9RdRlf*oGjinN1?yhF6XzA7V
    INFO: The password for user kibanaro is 7kZvau11cPn6Y1SbOsdr8Kwr*BRiK3u+
    INFO: The password for user logstash is SUbk4KTmLl*geQbUg0c5tyfwahjDMhx5
    INFO: The password for user readall is ?w*Itj1Lgz.5w.C7vOw0Kxi7G94G8bG*
    INFO: The password for user snapshotrestore is Z6UXgM8Sr0bfV.i*6yPPEUY3H6Du2rdz

>     WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, and Filebeat nodes if necessary, and restart the services.

Reviewer 2: ```suggestion
    WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, and Filebeat nodes if necessary.
```

   .. code-block:: console
      :class: output
      :emphasize-lines: 1,2,3,9,10

>       INFO: The new password for Filebeat is kwd139yG?YoIK?lRnqcXQ4R4gJDlAqKn

Reviewer 2: ```suggestion
      INFO: The new password for Filebeat is Wkw+b2rM6BEOwUmGfr*m*i1ithWw.dg2
```

Changing the passwords in a distributed environment
---------------------------------------------------

> Follow the instructions below to change the passwords for all Wazuh indexer users, Wazuh manager API users, and the Wazuh dashboard user.

Reviewer: The "Wazuh dashboard user" isn't evident from reading the introduction. Does it mean `wazuh-wui`? `admin`? `kibanaserver`? Other?

      INFO: The password for Wazuh API user wazuh is ivLOfmj7.jL6*7Ev?UJoFjrkGy9t6Je.
      INFO: The password for Wazuh API user wazuh-wui is fL+f?sFRPEv5pYRE559rqy9b6G4Z5pVi

>       INFO: The new password for Filebeat is kwd139yG?YoIK?lRnqcXQ4R4gJDlAqKn

Reviewer 2: ```suggestion
      INFO: The new password for Filebeat is Wkw+b2rM6BEOwUmGfr*m*i1ithWw.dg2
```

   .. note:: Repeat step 2 on `every Wazuh server node`.

#. On your `Wazuh dashboard node`, run the following command to update the `kibanaserver` password in the Wazuh dashboard keystore. Replace ``<KIBANASERVER_PASSWORD>`` with the random password generated in the first step.

   .. code-block:: console

      # curl -sO https://packages.wazuh.com/|WAZUH_CURRENT_MINOR|/wazuh-passwords-tool.sh
      # bash wazuh-passwords-tool.sh --user kibanaserver --password <KIBANASERVER_PASSWORD>
   
   .. code-block:: console
      :class: output

>       INFO: The password for the kibanaserver user in the dashboard has been updated to 'EKf49pm3QtqszKgWiz.HRfEc5adN7QFY' necessary, and restart the services.

Reviewer 2: ```suggestion
      INFO: The password for the kibanaserver user in the dashboard has been updated to 'EKf49pm3QtqszKgWiz.HRfEc5adN7QFY' necessary.
```

      INFO: The password for the kibanaserver user in the dashboard has been updated to 'EKf49pm3QtqszKgWiz.HRfEc5adN7QFY' necessary.

> #. Again, on your `Wazuh dashboard node`, run the following command to update the *wazuh-wui* password in the Wazuh dashboard keystore. Replace ``<WAZUH-WUI_PASSWORD>`` with the random password generated in the second step. Use the ``-A`` or ``--api`` option to change the password for the ``wazuh-wui`` user in the Wazuh dashboard node.

Reviewer: The second step generates more than one password. Do we mean `Wazuh API user wazuh-wui` password?

      # bash wazuh-passwords-tool.sh --api --user wazuh-wui --password <WAZUH-WUI_PASSWORD>

   .. code-block:: console
      :class: output

>       INFO: Updated wazuh-wui user password in wazuh dashboard to 'r7jH.SQ4SMqbzVXcbJrkiyrwvWd+G*w8'. Remember to restart the service.

Reviewer 2: ```suggestion
      INFO: Updated wazuh-wui user password in wazuh dashboard to 'r7jH.SQ4SMqbzVXcbJrkiyrwvWd+G*w8'.
```
